src/components/form/templates/types.ts

```diff
@@ -0,0 +1,68 @@
+// These are defined by react-jsonschema-form and added
+// here as interfaces for type hinting
```

Comment:
Shouldn't these interfaces be exported further? Or does that happen automatically?

Reply:
Also, is there any way we could make use of them in the examples?

Reply:
They should not be exposed to consumers if that's what you mean. They are only internal and used by the form component. If we one day find another library for rendering forms from a schema and decide to use that instead, we will be in big trouble if these are exposed :smile:

Reply:
Ah, ok. So these don't describe the schema format that our form-component takes, only the data it passes on to the lib we're using?

Reply:
Yeah, and the data we get from it as well :slightly_smiling_face:

Reply:
Ideally, these would have been defined in the library itself, but they are only described in the docs as far as I can see

Reply:
👍